pkg/blockdevice/probe/probe.go

```diff
@@ -60,15 +61,18 @@ func FileSystem(path string) (sb filesystem.SuperBlocker, err error) {
 	// Sleep for up to 5s to wait for kernel to create the necessary device files.
 	// If we dont sleep this becomes racy in that the device file does not exist
 	// and it will fail to open.
-	for i := 0; i <= 100; i++ {
+	err = retry.Constant(5 * time.Second).Retry(func() error {
```

Comment:
This says "retry every second (default unit) for  seconds."

Reply:
Given that we sleep 50ms before, we probably want to go with something lower than 1s? Probably insignificant in the long run, but not sure there's a need to change.